src/native/monodroid/pinvoke-override-api.cc

```diff
@@ -363,6 +357,15 @@ _monodroid_lookup_replacement_method_info (const char *jniSourceType, const char
 	return JniRemapping::lookup_replacement_method_info (jniSourceType, jniMethodName, jniMethodSignature);
 }
 
+static void
+monodroid_log_traces (uint32_t kind, const char *first_line)
```

Comment:
There's no mention of `monodroid_log_traces` in the commit message.  What's it _for_?

Reply:
There's no mention because it's not actually used here - this PR which sets up infrastructure for tracing.  The PR started this way (thus the branch name), but then I decided that it should only introduce "neutral" changes and prepare infrastructure for follow-up PRs which actually use the infrastructure (like Perfetto implementation)

It's part of code introduced in `src/native/monodroid/monodroid-tracing.cc`, also not really used by anything in this PR.